otter_welcome_buddy/common/constants.py

```diff
@@ -10,15 +10,15 @@ class CronExpressions(Enum):
 
 COMMAND_PREFIX: str = "!"
 
+BOT_TIMEZONE: str = "America/Mexico_City"
+
+FONT_PATH: str = "assets/fonts/Inter-Regular.ttf"
+
 DATA_DIR: str = "data"
 DATA_FILE: str = "otter-buddy.db"
 DATA_FILE_PATH: str = os.path.join(DATA_DIR, DATA_FILE)
 
-ALL_DIRS = (
-    attrib_value
-    for attrib_name, attrib_value in list(globals().items())
-    if attrib_name.endswith("DIR")
-)
+ALL_DIRS = (attrib_value for attrib_name, attrib_value in list(globals().items()) if attrib_name.endswith("DIR"))
```

Comment:
Why one liners? This makes my life harder to read this kind of code

Reply:
Because when I increased the line size it was updated automatically, not sure why is harder to read, 120 chars is more than enough imo unless you have a square monitor or use a letter size really big lol